consensus/polybft/state_test.go

```diff
@@ -300,17 +300,17 @@ func TestState_cleanCommitments(t *testing.T) {
 	assert.Equal(t, lastCommitmentToIndex, commitment.Message.ToIndex)
 
 	for i := uint64(1); i < numberOfCommitments; i++ {
-		c, err := state.getCommitmentMessage(i*stateSyncMainBundleSize + lastCommitmentToIndex - 1)
+		c, err := state.getCommitmentMessage(i*stateSyncCommitmentSize + lastCommitmentToIndex - 1)
 		assert.NoError(t, err)
 		assert.Nil(t, c)
 	}
 
-	bundles, err := state.getBundles(0, maxBundlesPerSprint)
+	bundles, err := state.getStateSyncProof(0)
 	assert.NoError(t, err)
 	assert.Nil(t, bundles)
 }
 
-func TestState_insertAndGetBundles(t *testing.T) {
+func TestState_insertAndGetStateSyncProof(t *testing.T) {
```

Comment:
Could we namespace the tests with something like `TestState_StateSync_XXX`? It would be easier to know what section they cover.

Reply:
Done in https://github.com/0xPolygon/polygon-edge/pull/1013/commits/7e8df2f49cb95d9fce492cb61862eed4c7c4b748